src/cplscheme/config/CouplingSchemeConfiguration.hpp

```diff
@@ -270,6 +270,9 @@ class CouplingSchemeConfiguration : public xml::XMLTag::Listener {
   void checkIfDataIsExchanged(
       DataID dataID) const;
 
+  // get waveform order for read-data of given name. Will raise an error, if data name is not found in the read-data of this participant.
```

Comment:
```suggestion
  /// Get waveform order for read data of given name. Raises an error if data name is not found in the read data of this participant.
```

Reply:
Done in b47a2b1ab5f7eb3a6761c23b3efb78893c16b706.